### PR TITLE
docs: Add VueSchool Summer Sale banner

### PR DESCRIPTION
**Pull request by nicodevs**
This PR adds the Vue School banner, similar to the previous [PR 848](https://github.com/vuejs/vue-router-next/pull/848).

[Visuals can be seen here.](https://postimg.cc/gallery/qvndjHD)

### CODE REVIEW

**Comment by posva** [comment]
When should this be deployed?

**Comment by nicodevs** [comment]
@posva, the landing page [is on production now](https://vueschool.io/sales/summer-vue/?friend=vuerouter) so you can merge this when you see fit :)

File: docs/.vitepress/theme/Layout.vue [hunk 1]
```diff
@@ -1,42 +1,63 @@
...
+          </a>
+        </div>
+      </template>
+    </ParentLayout>
+  </div>
 </template>
 
 <script>
 import DefaultTheme from 'vitepress/dist/client/theme-default'
+import BannerTop from '../components/BannerTop.vue'
```
Comment (posva) [inline docs/.vitepress/theme/Layout.vue:37]: Can you make this async?

**Review by posva — CHANGES_REQUESTED** [review]

File: docs/.vitepress/theme/Layout.vue (hunk 1 above)
Comment (nicodevs) [reply docs/.vitepress/theme/Layout.vue]: Done! Let me know if it's OK @posva

**Review by posva — APPROVED** [review]
Thanks!

**Pull request merged**